lib/process-services-cloud/src/lib/form/components/widgets/data-table/data-table-adapter.widget.ts

```diff
@@ -44,6 +46,12 @@ export class WidgetDataTableAdapter extends ObjectDataTableAdapter {
         return this.hasAllColumnsLinkedToData() && this.hasAllMandatoryColumnPropertiesHaveValues();
     }
 
+    private hideColumnTypes(typesToHide: string[]): void {
+        this.columns
+            .filter(column => typesToHide.includes(column.type))
+            .forEach(column => column.isHidden = true);
```

Comment:
i think instead of setting isHidden property for some columns, we can just filter them out at all (i guess with this solution once we complete the form, our payload will still contain hidden columns which makes no sense but im not sure, have you tested it?)
something like 
```suggestion
            super.getColumns().filter....
```

Reply:
PR will be closed